cmd/pipelines-as-code-controller/main.go

```diff
@@ -28,5 +32,30 @@ func main() {
 		log.Fatal("failed to init kinit client : ", err)
 	}
 
+	loggerConfiguratorOpt := evadapter.WithLoggerConfiguratorConfigMapName(logging.ConfigMapName())
+	loggerConfigurator := evadapter.NewLoggerConfiguratorFromConfigMap(PACControllerLogKey, loggerConfiguratorOpt)
+	copt := evadapter.WithLoggerConfigurator(loggerConfigurator)
+	// put logger configurator to ctx
+	ctx = evadapter.WithConfiguratorOptions(ctx, []evadapter.ConfiguratorOption{copt})
+	// set up kubernetes interface to retrieve configmap with log configuration
+	ctx = context.WithValue(ctx, client.Key{}, run.Clients.Kube)
+
+	ns, err := getPacNamespace()
+	if err != nil {
+		log.Fatal("failed to get current pac namespace : ", err)
+	}
```

Comment:
```suggestion
	ns := os.Getenv("SYSTEM_NAMESPACE")
```
We can relay on `SYSTEM_NAMESPACE` env